README.md

```diff
@@ -16,8 +26,18 @@ Be sure to install node modules:
 npm install
 ```
 
-It is no ready to start a local development environment:
+It is now ready to start a local development environment:
 
 ```shell
 npm start
 ```
+
+## Authors
+
+- **Kyle Shevlin** - _Initial work_ - [Twitter](https://twitter.com/kyleshevlin)
+
+<!-- See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project. -->
+<!--
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details -->
```

Comment:
Uncomment this, make it.

```
## License

This project is licensed under the MIT License
```

We can indicate there's a file when the file is added to the repo. FWIW, I don't love having commented out code sitting around the app anywhere, so if you want to add something, my philosophy will be to add what can be added now, make another commit and update later when you have everything.